specs/data/overwrite/april/users.csv

```diff
@@ -0,0 +1,4 @@
+id,test type
+1,node app.js  --files specs/data/overwrite/march/users.csv
```

Comment:
This test data is incredibly confusing. I wouldn't put `node app.js ...` as my test data. I also wouldn't put file paths as my test data, especially if they are the same as the paths for my test!